### PR TITLE
Add Tier 2 Testing

### LINKED ISSUE #626: Add automated (tier 2) testing for long running tests
In particular 'test test_release' and 'test test_many_blocks".

### PR DESCRIPTION
**Pull request by robinbb**
Fixes #626

### CODE REVIEW

**Comment by trevorbernard** [comment]
https://buildkite.com/granola/mina-indexer/builds/721#018ea225-401c-4c41-8a9d-4c078b511ed8/30-419

The ingest-all script needs updating

File: ops/buildkite-pipeline.yaml [hunk 1]
```diff
@@ -1,22 +1,43 @@
...
+  depends_on:
+  - build
+  agents:
+    mina-log-storage: true
+    docker: true
+  command: |
+    nix develop --command just build
+    nix develop --command tests/regression test_many_blocks
+    # TODO: why is this not working?
+    # nix develop --command tests/regression test_release
```
Comment (Isaac-DeFrain) [inline ops/buildkite-pipeline.yaml:25]: Did you want to remove this before merging?

**Review by Isaac-DeFrain — APPROVED** [review]
Great work!

File: ops/buildkite-pipeline.yaml (hunk 1 above)
Comment (robinbb) [reply ops/buildkite-pipeline.yaml:25]: Is it okay to keep it? It documents the fact that a test is failing.

**Pull request merged**